t/ui/06-operator_links.t

```diff
@@ -35,59 +35,48 @@ unless ($driver) {
 my $t = Test::Mojo->new('OpenQA::WebAPI');
 
 # we don't want to test javascript here, so we just test the javascript code
-# List with no login
+note 'List with no login';
 $t->get_ok('/tests')->status_is(200)->content_like(qr/is_operator = false;/, "test list rendered without is_operator");
-
-# List with an authorized user
+note 'List with an authorized user';
 $test_case->login($t, 'percival');
 $t->get_ok('/tests')->status_is(200)->content_like(qr/is_operator = true;/, "test list rendered with is_operator");
-
-# List with a not authorized user
+note 'List with a not authorized user';
 $test_case->login($t, 'lancelot', email => 'lancelot@example.com');
 $t->get_ok('/tests')->status_is(200)->content_like(qr/is_operator = false;/, "test list rendered without is_operator");
-
-# now the same for scheduled jobs
+note 'now the same for scheduled jobs';
 $t->delete_ok('/logout')->status_is(302);
-
-# List with no login (absence of cancel button already checked in 01-list.t)
+note 'List with no login (absence of cancel button already checked in 01-list.t)';
 $t->get_ok('/tests')->status_is(200);
-
-# List with an authorized user (presence of cancel button already checked in 01-list.t)
+note 'List with an authorized user (presence of cancel button already checked in 01-list.t)';
 $test_case->login($t, 'percival');
 $t->get_ok('/tests')->status_is(200);
-
-# List with a not authorized user
+note 'List with a not authorized user';
 $test_case->login($t, 'lancelot', email => 'lancelot@example.com');
 $t->get_ok('/tests')->status_is(200);
-
-# operator has access to part of admin menu - using phantomjs
+note 'operator has access to part of admin menu';
 $driver->title_is("openQA", "on main page");
 $driver->find_element_by_link_text('Login')->click();
-# we're back on the main page
+note 'we are back on the main page';
 $driver->title_is("openQA", "back on main page");
-# but ...
-
+note 'but ...';
```

Comment:
If you're changing things here please improve the phrasing.

Reply:
done